hpx/util/annotated_function.hpp

```diff
@@ -83,7 +83,7 @@ namespace hpx { namespace util
             /* update the task wrapper in APEX to use the specified name */
             threads::set_self_timer_data(
                 external_timer::update_task(threads::get_self_timer_data(),
-                desc_));
+                std::string(name)));
```

Comment:
This needs a `#include <string>` (https://144987-4455628-gh.circle-artifacts.com/0/hpx_inspect_report.html).